### ISSUE TITLE
Add the dynamic element to the shortcode & shortcode wrapper element

### ISSUE THREAD

**Issue by Schokolader** [soflyy/oxygen-bugs-and-features#2908]
**Describe the feature you'd like to see included in Oxygen.**
It would be awesome to have the shortcode element to be filled dynamically   

**What are the use cases for this feature?**
You could add a shortcode to a custom field for example a shortcode for a gallery from the modula gallery plugin. 
In your single blog post template you could add the shortcode element mapped dynamically to the shortcode of the gallery. 
So it would show the gallery from your custom field shortcode. 

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*
There are many other usecases. You could install a audio player plugin and add the shortcodes to your posts in custom fields. 
Basically every plugin that uses shortcodes to execute could be easily integrated into a template dynamically.
